src/test/java/com/sforce/cd/apexUnit/fileReader/ApexManifestFileReaderTest.java

```diff
@@ -155,4 +157,20 @@ public void cleanUpTestFiles() {
 			LOG.info("Test files directory does not exist; hence not deleted");
 		}
 	}
+
+	@Test
+	public void namespaceParserTest() {
```

Comment:
You have a positive test case here. What about some negative test cases? Test cases where you are going to create a failure? 
stringWithNamespaceAndClassname = "1.2.3"
stringWithNamespaceAndClassname = "..."
stringWithNamespaceAndClassname = "1."
stringWithNamespaceAndClassname = ".1"